### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Features added in this PR:
- Added dark mode

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import "./App.css";
 import logo from "./logo.png";
-import { Navbar } from "./note-components/Navbar/navbar";
+// import { Navbar } from "./note-components/Navbar/navbar";
```
Comment (Jia-A) [inline src/App.js:3]: You can remove this commented code.

File: src/note-components/Authentication/auth.css [hunk 2]
```diff
@@ -80,4 +67,8 @@
 
 .btn-no-bg-color:active {
     color: var(--accent-hover);
```
Comment (Jia-A) [inline src/note-components/Authentication/auth.css:69]: Good that you used CSS variables.

**Review by Jia-A — APPROVED** [review]
Everything looks fine.
Good implementation of theme logic.
Formatted code is also nice.
Good naming conventions. You can merge it.

**Review by srishtigautam21 — APPROVED** [review]
The code looks clean and understandable
Good work
Good to merge

File: src/note-components/Homepage/homepage.jsx [hunk 3]
```diff
@@ -83,23 +80,23 @@ return (
           className="note-area text-color"
           onChange={(e)=> setNote(() => ({...notes, mainContent: e.target.value, currentDate: new Date().toLocaleString()}))}></textarea>
         <div className="note-footer">
-          <select name="tags" onClick={(e)=> setNote(() => ({...notes, tags: e.target.value}))}>
+          <select name="tags" className="select" onClick={(e)=> setNote(() => ({...notes, tags: e.target.value}))}>
             <option selected disabled>Tags</option>
             {tagItem.map((tagItemName) =>
             <option value={tagItemName}>{tagItemName}</option>
```
Comment (purvasharmadev) [inline src/note-components/Homepage/homepage.jsx:86]: Fix: Add key props to avoid warnings

**Review by purvasharmadev — APPROVED** [review]
code is well structured and readable, good to merge.

File: src/App.js (hunk 1 above)
Comment (Megha-Joshi) [reply src/App.js:3]: Okay

File: src/note-components/Homepage/homepage.jsx (hunk 3 above)
Comment (Megha-Joshi) [reply src/note-components/Homepage/homepage.jsx:86]: Okay, will do it

**Pull request merged**